hudi-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadata.java

```diff
@@ -117,172 +93,15 @@ public HoodieBackedTableMetadata(Configuration conf, String datasetBasePath, Str
     } else {
       LOG.info("Metadata table is disabled.");
     }
-
```

Comment:
I assume all of this code, is just verbatim moved up to the base class ?

Reply:
Correct